packages/sdk/src/utils.ts

```diff
@@ -93,3 +95,33 @@ export const BatchReadWithErrors = (multicall2: Multicall2) => (contract: Contra
     })
   );
 };
+
+/**
+ * @notice Return average block-time for a period.
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export async function averageBlockTimeSeconds(lookbackSeconds?: number, networkId?: number): Promise<number> {
```

Comment:
is this copied from common? Can we just import it? Isn't common FE compatible and already used in the SDK?

Reply:
its copied from common. i can try, i dont think its used in sdk currently but i could be wrong

Reply:
i updated this to import the function from common, seems to work

Reply:
i had to copy and paste the functions again due to build failing in across frontend when common was a dep